src/features/dental.feature

```diff
@@ -1,6 +1,44 @@
 Feature: Test for Dental page
 
-  Scenario: Open Dental Insurance
+  Scenario: View Dental header elements 
     Given the user is on the Dental page
     When the Dental Insurance product page is opened
-    Then the main Dental title and description should be correct
\ No newline at end of file
+    Then the Dental title and description should be correct
+    And user can see Offerings link
+    And user can see Healthcare link
+    And find Offerings link is working
+    Then the Offerings page is opened
+    Then the user is on the Dental page
```

Comment:
Missing step, or above mentioned step name is misleading, there is no action indicating user got back to dental page, open page step should only be first step in scenario